src/components/NavBar/NavBar.jsx

```diff
@@ -27,24 +30,35 @@ const links = [
 ];
 
 export default function NavBar() {
+  const [showNavLinks, setShowNavLinks] = useState(false);
   const { pathname } = useRouter();
   const isActive = (path, to) => {
     if (path.split('/')[1] === to.split('/')[1]) {
       return true;
     }
     return false;
   };
+  const toggleMenuLink = (value) => {
+    document.body.style.overflow = value ? 'hidden' : 'auto';
```

Comment:
As Dustin said before accessing dom is expensive in React! We should find another way to do it